src/game/spell/Cheat.cpp

```diff
@@ -85,7 +85,7 @@ void CheatDrawText() {
 		for(size_t i = 0; i < sp_max_ch.length(); i++) {
 			Vec2f d = p + Vec2f(sizX * i, sp_max_y[i]);
 			
-			sp_max_y[i] = std::sin(d.x + arxtime.now_f() * ( 1.0f / 100 )) * 30.f * modi;
+			sp_max_y[i] = std::sin(d.x + (float)platform::getTimeMs() * ( 1.0f / 100 )) * 30.f * modi;
```

Comment:
I don't think casting absolute platform time values to float is a good idea - they can be arbitrarily large, meaning we may lose precision here. Really, absolute platform time should only ever be used to calculate time differences and then do something with those since the absolute values have no meaning, only the rate at which they increase.